*Added span with total-kp-score class to display users total points

rehazler opened the pull request:
*Added total-kp-scores class to format the users total points
*Created function to display the users total points in above the progress circle
*Changed point addition to increase progress circle position based on new point values rather than starting over
*Added level caps for users to simulate leveling system
*Decreased interval for the progress circle to complete total animation to decrease over all user time consumption for higher point values

rehazler committed:
*Added span with total-kp-score class to display users total points
*Added total-kp-scores class to format the users total points
*Created function to display the users total points in above the progress circle
*Changed point addition to increase progress circle position based on new point values rather than starting over
*Added level caps for users to simulate leveling system
*Decreased interval for the progress circle to complete total animation to decrease over all user time consumption for higher point values

rehazler committed:
Removed unused function

bitmogram reviewed (commented):
Everything looks good as far as I can tell.

rehazler committed:
*Fixed merge conflicts
*Added subtraction animation
*Fixed interval timing that allowed for overlapping and conflicting intervals
